cmd/skywire-visor/commands/root.go

```diff
@@ -262,6 +268,33 @@ func runVisor(conf *visorconfig.V1) {
 			conf.Hypervisors = append(conf.Hypervisors, pubkey)
 		}
 	}
+	//autopeering should only happen when there is no local or remote hypervisor set in the config.
+	if conf.Hypervisors != nil {
```

Comment:
Can be one line

Reply:
Also what is this check even doing? Its false by default and if the user sets it to true, we will overwrite what we do here anyways?

Reply:
somethingI stareted implementing in one way and then thought better of it.

I've removed the vestigial logging